test/jdk/sun/security/pkcs11/Signature/SignatureTestPSS.java

```diff
@@ -62,82 +62,57 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public void main(Provider p) throws Exception {
-        Signature sig;
-        try {
-            sig = Signature.getInstance(SIGALG, p);
-        } catch (NoSuchAlgorithmException e) {
+        if (!PSSUtil.isSignatureSupported(p)) {
             System.out.println("Skip testing RSASSA-PSS" +
```

Comment:
throw SkippedException here.

Reply:
Ok. I initially thought that the SkippedException only applies to library not found and/or un-configured OS. With this extension of the meaning of SkippedException, many of the existing tests have to be updated to match.

Reply:
Yes, I did #1 update with [JDK-8313206](https://bugs.openjdk.org/browse/JDK-8313206) and plan to do update more scenarios with [JDK-8313575](https://bugs.openjdk.org/browse/JDK-8313575)